demos/usage-context-menu.py

```diff
@@ -0,0 +1,149 @@
+import dash_core_components as dcc
+import time
+
+app = dash.Dash(__name__)
+app.layout = html.Div(
+    [
+        cyto.Cytoscape(
+            id="cytoscape",
+            elements=[
+                {
+                    "data": {"id": "one"},
+                    "position": {"x": 50, "y": 50},
+                },
+                {
+                    "data": {"id": "two"},
+                    "position": {"x": 200, "y": 200},
+                },
+                {"data": {"source": "one", "target": "two"}},
+            ],
+            layout={"name": "preset"},
+            contextMenu=[
+                {
+                    "id": "add-node",
+                    "content": "Add Node (JS)",
+                    "tooltipText": "Add Node",
+                    "coreAsWell": "true",
+                    "onClick": "add_node",
+                },
+                {
+                    "id": "remove",
+                    "content": "Remove (JS)",
+                    "tooltipText": "Remove",
+                    "selector": "node, edge",
+                    "onClick": "remove",
+                },
+                {
+                    "id": "add-edge",
+                    "content": "Add Edge (JS)",
+                    "tooltipText": "add edge",
+                    "selector": "node",
+                    "onClick": "add_edge",
+                },
+                {
+                    "id": "split-edge",
+                    "content": "Split Edge (PY)",
+                    "tooltipText": "split edge",
+                    "selector": "edge",
+                },
+                {
+                    "id": "revert-edge",
+                    "content": "Revert Edge (PY)",
+                    "tooltipText": "revert edge",
+                    "selector": "edge",
+                },
+                {
+                    "id": "add-2-nodes",
+                    "content": "Add 2 Nodes (NS)",
+                    "tooltipText": "add 2 nodes",
+                    "coreAsWell": "true",
+                    "onClickUser": "add_2_nodes",
+                },
+            ],
+            stylesheet=[
+                {
+                    "selector": "edge",
+                    "style": {
+                        "target-arrow-color": "grey",
+                        "target-arrow-shape": "vee",
+                        "curve-style": "straight",
+                    },
+                },
+                {"selector": "node", "style": {"label": "data(label)"}},
+            ],
+        ),
+        html.Div(id="output-selection"),
+        html.Div(id="output"),
+    ]
+)
+
+
+@callback(
+    Output("output", "children"),
+    Output("cytoscape", "elements"),
+    Input("cytoscape", "contextMenuData"),
+    State("cytoscape", "elements"),
+    State("cytoscape", "selectedEdgeData"),
+    State("cytoscape", "selectedNodeData"),
+)
+def update_output(ctx, elements, selectedEdges, selectedNodes):
```

Comment:
This is super minor, but maybe give the variable a different name than `ctx` since that's the name used in the docs for callback context.